src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/DefendantResponseStaffNotificationService.java

```diff
@@ -61,19 +73,40 @@ public void notifyStaffDefenceSubmittedFor(
         );
     }
 
+    private boolean isFullAdmission(Claim claim) {
+        ResponseType responseType = claim.getResponse().orElseThrow(IllegalArgumentException::new).getResponseType();
+        return responseType == ResponseType.FULL_ADMISSION;
+    }
+
     public static Map<String, Object> wrapInMap(
         Claim claim,
         String defendantEmail
     ) {
         Map<String, Object> map = new HashMap<>();
+
+        Response response = claim.getResponse().orElseThrow(IllegalStateException::new);
         map.put("claim", claim);
-        map.put("response", claim.getResponse().orElseThrow(IllegalStateException::new));
+        map.put("response", response);
         map.put("defendantEmail", defendantEmail);
-        map.put("defendantMobilePhone", claim.getResponse()
-            .orElseThrow(IllegalStateException::new)
-            .getDefendant()
-            .getMobilePhone()
-            .orElse(null));
+
+        switch (response.getResponseType()) {
+            case FULL_DEFENCE:
+                map.put("defendantMobilePhone", response
+                    .getDefendant()
+                    .getMobilePhone()
+                    .orElse(null));
+                break;
+            case FULL_ADMISSION:
+                FullAdmissionResponse fullAdmissionResponse = (FullAdmissionResponse) response;
+                map.put("paymentOption", fullAdmissionResponse.getPaymentOption());
+                map.put("paymentOptionDescription", fullAdmissionResponse
+                    .getPaymentOption().getDescription().toLowerCase());
+                map.put("responseDeadline", formatDate(claim.getResponseDeadline()));
```

Comment:
A `responseDeadline` and `fourteenDaysFromNow` should be outside the switch statement IMO regardless they are just used in full admission flow atm.

Reply:
these are not required in the email template for full defence. only required for full admission email template

Reply:
Explained my position above. I am happy to compromise here on the assumption that content providers are considered tech debt.